pom.xml

```diff
@@ -1,90 +1,56 @@
 <?xml version="1.0" encoding="utf-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
     <modelVersion>4.0.0</modelVersion>
     <groupId>au.org.emii</groupId>
     <artifactId>geoserver</artifactId>
-    <packaging>war</packaging>
-    <version>2.6.2</version>
+    <packaging>pom</packaging>
+    <version>1.0.0</version>
```

Comment:
You know what I think about the choice of version here.

I personally think it should reflect somehow the version of Geoserver that's encapsulated in the build.